tests/cluster-users-test/sar-test.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash -e
+
+# Description: This script performs a SelfSubjectAccessReview test by asking the k8s apiserver what permissions does each user have
+# Tested users: admin@kyma.cx, developer@kyma.cx, user@kyma.cx
+# Required ENVS: 
+#  - EMAIL_FILE: path to a file with the email address of the user (used as username)
+#  - PASSWORD_FILE: path to a file with the password for the user
+#  - NAMESPACE: namespace in which we perform the tests
+
+function testPermissions() {
+	USER="$1"
+	OPERATION="$2"
+	RESOURCE="$3"
+	EXPECTED="$4"
+	set +e
+	TEST=$(kubectl auth can-i "${OPERATION}" "${RESOURCE}" --as "${USER} -n ${NAMESPACE}")
+	set -e
+	if [[ "${TEST}" == "${EXPECTED}" ]]; then
+		echo "----> PASSED"
+		return 0
+	fi
+	echo "----> |FAIL| Expected result: ${EXPECTED} vs Actual result: ${TEST}"
+	return 1
+}
+
+function runTests() {
+	echo "--> developer@kyma.cx should be able to get Deployments"
```

Comment:
Can you add tests showing that the developer can access namespaced resources (pods, deployments) in the namespace he has rolebinding and cannot modify system namespace like (kube-system, kyma-system, istio-system)?

Reply:
Implemented, please verify